docs/api.md

```diff
@@ -1637,6 +1638,55 @@ Example `options` argument:
 options.mode can have 3 values: 0 (SEQUENCE), 1 (AUTO), 2 (REDSTONE)
 All options attributes are false by default, except mode which is 2 (as to replicate the default command block in Minecraft).
 
+#### bot.supportFeature(name)
+
+This can be used to check is a specific feature is available in the current Minecraft version. This is usually only required for handling version-specific functionality.
+
+Available Features:
```

Comment:
I think it's best if we put a link to the json file here and don't copy paste its content in the doc (that would increase maintenance chores too much)

Reply:
Fixed